### ISSUE TITLE
Predict bounding boxes of the different objects for test set

### ISSUE THREAD

**Issue by akhanss** [yhenon/pytorch-retinanet#26]
Thanks for this great work. I really appreciate your efforts. However, I am just wondering if you can describe a brief procedure to predict bounding boxes of the different objects for a held out test set (having only images, no annotation at all) using the trained model. We used CSV to train the model.

**Comment by MSoulPlayer** [comment]
@akhanss see 
[visualize.py](https://github.com/yhenon/pytorch-retinanet/blob/master/visualize.py)
change the --csv_val para to the test CSV file with the format img.jpg,,,,,

**Comment by akhanss** [comment]
Thanks a lot @MSoulPlayer for your kind reply!

**Comment by yhenon** [comment]
Indeed. Note that I'm planning a way to visualize images without passing an annotation file, instead just by providing a directory containing images.
